integreat_cms/cms/templatetags/content_filters.py

```diff
@@ -83,6 +83,28 @@ def get_language(language_slug):
     return Language.objects.filter(slug=language_slug).first()
 
 
+@register.simple_tag
+def minor_edit_label(region, language):
+    """
+    This tag returns the label of the minor edit field of the given form
+
+    :param region: current region
+    :type region: ~integreat_cms.cms.models.regions.region.Region
+
+    :param language: The current language
+    :type language: ~integreat_cms.cms.models.languages.language.Language
+
+    :return: The minor edit label
+    :rtype: str
+    """
+    language_node = region.language_node_by_slug[language.slug]
+    if language_node.is_leaf():
+        return _("Implications on the translation status")
+    if language_node.is_root():
+        return _("Implications on other translations")
+    return _("Implications on the status of this and other translations")
```

Comment:
Is this case really necessary. Do we have some stand-alone language nodes?

Reply:
Not in the production database, but our whole model is built on the assumption that we might have language tree nodes that are neither root nodes nor leaf nodes.